native/acter/api.rsh

```diff
@@ -1959,10 +1956,27 @@ object Comment {
     /// create a draft builder to reply to this comment
     fn reply_builder() -> CommentDraft;
 
+    /// update builder
+    fn update_builder() -> Result<CommentUpdateBuilder>;
+
+    /// replace the current task with one with the latest state
+    fn refresh() -> Future<Result<Comment>>;
+
     /// whether or not this user can redact this item
     fn can_redact() -> Future<Result<bool>>;
 }
 
+object CommentUpdateBuilder {
+    /// set description text
+    fn content_text(body: string);
+
+    /// set description html text
+    fn content_formatted(body: string, html_body: string);
+
+    /// send builder update
+    fn send() -> Future<Result<EventId>>;
+}
+
```

Comment:
Our UI currently doesn't have any notion of updating comments and having updated comments (like we don't have notifications nor activities for those either), I am not sure we want to expose this API yet...

Reply:
Fixed by https://github.com/acterglobal/a3/pull/3005/commits/2c274fde939abf4b3d5411880b6d897be630e2ba